src/gui/editors/SongEditor.cpp

```diff
@@ -170,9 +186,9 @@ SongEditor::SongEditor( Song * song ) :
 	connect( m_masterVolumeSlider, SIGNAL(sliderReleased()), this,
 			SLOT(hideMasterVolumeFloat()));
 
-	m_mvsStatus = new TextFloat;
-	m_mvsStatus->setTitle( tr( "Master volume" ) );
-	m_mvsStatus->setPixmap( embed::getIconPixmap( "master_volume" ) );
+	m_mvsStatus	= new TextFloat;
+	m_mvsStatus->setTitle(tr("Master volume"));
+	m_mvsStatus->setPixmap(embed::getIconPixmap("master_volume"));
```

Comment:
This is not a show-stopper (I want the code to look nice too) ... But in the future try to no edit lines when you don't actually need change them, because changes may introduce merge conflicts for other people, so try to keep the PR to the point.

I usually run `git gui` before committing and that allows me to see where I have added empty lines or changed whitespace etc for no good reason, and I can right click on those lines and select "revert line".

When doing follow-up commits to a PR like you are doing now, that :arrow_up: becomes harder to do. Then manual editing is perhaps your quickest option.

Reply:
I try to. I have not knowingly changed it. I'll try to be more careful in the future 😄